history.go

```diff
@@ -44,6 +48,7 @@ func RunCommand(entrypoint string, args []string) (string, error) {
 func Run() error {
 	fmt.Println("Welcome to history")
```

Comment:
We don't want to print to the terminal when running tests, for example, so again it'll be convenient to inject a writer to write to. In the tests, `ioutil.Discard`. In production, `os.Stdout`.

Reply:
what if I just move to the CLI? I've already learned how to handle outputs in tests.